src/TerraformInitCommand.groovy

```diff
@@ -72,29 +68,11 @@ class TerraformInitCommand implements Resettable {
         return pieces.join(' ')
     }
 
-    private applyPluginsOnce() {
-        def remainingPlugins = globalPlugins - appliedPlugins
-
-        for (TerraformInitCommandPlugin plugin in remainingPlugins) {
-            plugin.apply(this)
-            appliedPlugins << plugin
-        }
-    }
-
-    public static void addPlugin(TerraformInitCommandPlugin plugin) {
-        this.globalPlugins << plugin
-    }
-
     public static TerraformInitCommand instanceFor(String environment) {
         return new TerraformInitCommand(environment)
     }
 
-    public static getPlugins() {
-        return this.globalPlugins
-    }
-
-    public static reset() {
-        globalPlugins = []
-        // This is awkward - what about the applied plugins...?
+    public String getEnvironment() {
+        return environment
```

Comment:
Did this need to be added?

Reply:
No, I must have put that in by accidetnt.